phoenix-core/src/main/java/org/apache/phoenix/schema/types/PhoenixArray.java

```diff
@@ -561,6 +666,23 @@ public int estimateByteSize(int pos) {
 		}
 		
 		@Override
+		public String toString() {
+			StringBuilder sb = new StringBuilder(TO_STRING_BEGIN);
+			boolean isFirst = true;
+			final Format defaultFormat = null;
+			for (Byte o : byteArr) {
+				if (isFirst) {
+					isFirst = false;
+				} else {
+					sb.append(TO_STRING_SEPARATOR);
+				}
+				sb.append(this.baseType.toStringLiteral(o, defaultFormat));
```

Comment:
Looks like the defaultFormat is always null. How about having a new method toStringLiteral(o) which delegates to toStringLiteral(o, defaultFormat) where defaultFormat is null?